includes/wizards/class-seo-wizard.php

```diff
@@ -70,7 +70,122 @@ public function get_length() {
 	/**
 	 * Register the endpoints needed for the wizard screens.
 	 */
-	public function register_api_endpoints() {}
+	public function register_api_endpoints() {
+		register_rest_route(
+			'newspack/v1/wizard/' . $this->slug,
+			'settings',
+			[
+				'methods'             => \WP_REST_Server::READABLE,
+				'callback'            => [ $this, 'api_get_seo_settings' ],
+				'permission_callback' => [ $this, 'api_permissions_check' ],
+			]
+		);
+		register_rest_route(
+			'newspack/v1/wizard/' . $this->slug,
+			'settings',
+			[
+				'methods'             => \WP_REST_Server::EDITABLE,
+				'callback'            => [ $this, 'api_update_seo_settings' ],
+				'permission_callback' => [ $this, 'api_permissions_check' ],
+				'args'                => [
+					'under_construction' => [
+						'sanitize_callback' => 'absint',
+					],
+					'separator'          => [
+						'sanitize_callback' => 'sanitize_text_field',
+					],
+					'verification'       => [],
+					'urls'               => [],
+				],
+			]
+		);
+	}
+
+	/**
+	 * API endpoint to retrieve all settings necessary to render the SEO wizard.
+	 *
+	 * @return WP_REST_Response with the info.
+	 */
+	public function api_get_seo_settings() {
+		$response = $this->get_seo_settings();
+		return rest_ensure_response( $response );
+	}
+
+	/**
+	 * Update SEO wizard settings.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response with the info.
+	 */
+	public function api_update_seo_settings( $request ) {
+		$cm = Configuration_Managers::configuration_manager_class_for_plugin_slug( 'wordpress_seo' );
```

Comment:
Maybe add a quick `is_configured()` check here and return an error if Yoast is not active?

Reply:
Whoops! https://github.com/Automattic/newspack-plugin/pull/435/commits/66c674360bfcfe7fa5f3d16282874cdb190ebd80

Reply:
The configuration managers generally have an `is_configured` check in every method that interacts with the external plugin, so we probably don't need an additional one right when they are instantiated. The fix in https://github.com/Automattic/newspack-plugin/pull/435/commits/66c674360bfcfe7fa5f3d16282874cdb190ebd80 addresses this. If you strongly feel we need an additional one, though, I'm happy to add.

Reply:
All good as long as it's checking somewhere :)